sentry-spring-boot/src/main/java/io/sentry/spring/boot/SentryWebfluxAutoConfiguration.java

```diff
@@ -19,6 +20,7 @@
 import reactor.core.scheduler.Schedulers;
 
 /** Configures Sentry integration for Spring Webflux and Project Reactor. */
+@AutoConfiguration
```

Comment:
I think this should be kept as is, since `AutoConfiguration` was only introduced in 2.7.0 of Spring Boot and as far as I can see the commit you linked in the issue (https://github.com/spring-projects/spring-boot/commit/08022ba86efa294f77574bda07851901b6c2b520) only seems to affect Spring Boot 3 but not Spring Boot 2. So I'd keep non jakarta files as they are.

Reply:
@adinauer 
thanks for your comments :)
and you're absolutely right, it's only affect spring boot above 3.*
so i reverted non-jakarta files :)